scripts/install.ps1

```diff
@@ -0,0 +1,463 @@
+
+    $httpClient = New-Object System.Net.Http.HttpClient($handler)
+    $userAgentHeader = New-Object System.Net.Http.Headers.ProductInfoHeaderValue("otelcol-sumo-installer", "0.1")
+    $httpClient.DefaultRequestHeaders.UserAgent.Add($userAgentHeader)
+
+    # set http client timeout to 30 seconds
+    $httpClient.Timeout = New-Object System.TimeSpan(0, 0, 30)
+
+    Write-Host "Getting installed version..."
+    $installedVersion = Get-InstalledVersion
+    $installedVersionStr = "none"
+    if ($installedVersion -ne $null) {
+        $installedVersionStr = $installedVersion
+    }
+    Write-Host "Installed version:`t${installedVersionStr}"
+
+    # Get versions, but ignore errors are we fallback to other methods later
+    Write-Host "Getting versions..."
+    $versions = Get-Version -Command All -HttpClient $httpClient
+    
+    # Use user's version if set, otherwise get latest version from API (or website)
+    if ($Version -eq "") {
+        if ($versions.Count -gt 0) {
+            $Version = $versions[0]
+        } else {
+            $Version = Get-Version -Command Latest -HttpClient $httpClient
+        }
+    }
+
+    $versionRegex = '(\d)\.(\d+)\.(\d+)(.*(\d+))'
+    $Matches = [Regex]::Matches($Version, $versionRegex)
+    $majorVersion = $Matches[0].Groups[1].Value
+    $minorVersion = $Matches[0].Groups[2].Value
+    $patchVersion = $Matches[0].Groups[3].Value
+    $suffix = $Matches[0].Groups[4].Value
+    $buildVersion = $Matches[0].Groups[5].Value
+    $productVersion = "${majorVersion}.${minorVersion}.${patchVersion}.${buildVersion}"
+
+    Write-Host "Version to install:`t${Version}"
+
+    # Check if otelcol is already in newest version
+    if ($installedVersion -eq $Version) {
+        Write-Host "OpenTelemetry collector is already in newest (${Version}) version"
+    } else {
+        # add newline before breaking changes and changelog
+        Write-Host ""
+        if (($installedVersion -ne "") -And ($versions -ne $null)) {
+            # Take versions from installed up to the newest
+            $index = $versions.IndexOf($installedVersion)
+            if ($index -gt 0) {
+                $changelog = Get-Changelog $httpClient
+                Show-BreakingChanges $versions[0..($index-1)] $changelog
+            }
+        }
+    }
+
+    Write-Host "Changelog:`t`thttps://github.com/SumoLogic/sumologic-otel-collector/blob/main/CHANGELOG.md"
+    # add newline after breaking changes and changelog
+    Write-Host ""
+
+    # Download MSI
+    $msiLanguage = "en-US"
+    $msiFileName = "otelcol-sumo_${productVersion}_${msiLanguage}.${archName}.msi"
+    $msiUri = "https://github.com/SumoLogic/sumologic-otel-collector/releases/download/"
+    $msiUri += "v${Version}/${msiFileName}"
+    $msiPath = "${env:TEMP}\${msiFileName}"
+    Get-BinaryFromUri $msiUri -Path $msiPath -HttpClient $httpClient
+
+    # Install MSI
+    [string[]] $msiProperties = @()
+    $msiProperties += "INSTALLATIONTOKEN=${InstallationToken}"
```

Comment:
Does it work for `v0.72.0` release? As I changed the name recently I would expect it to work from `v0.73.0` :thinking:

Reply:
Yes, it worked when I was testing it and it looks like it was included in `v0.72.0` https://github.com/SumoLogic/sumologic-otel-collector/blob/v0.72.0-sumo-0/packaging/msi/wix/package.wxs

Reply:
eh, seems that I unnecessary added support for `INSTALLTOKEN` again :roll_eyes: